OpenRA.Mods.Common/Traits/Player/ProvidesPrerequisite.cs

```diff
@@ -76,7 +86,9 @@ void Update(Player owner, string faction)
 				enabled = info.Factions.Contains(faction);
 
 			if (info.RequiresPrerequisites.Any() && enabled)
-				enabled = owner.PlayerActor.Trait<TechTree>().HasPrerequisites(info.RequiresPrerequisites);
+				enabled = techTree.HasPrerequisites(info.RequiresPrerequisites);
```

Comment:
Actually, I don't even use `RequiresPrerequisites` in my testcase, so I doubt this will be the cause.